bookkeeper-server/src/test/java/org/apache/bookkeeper/replication/TestReplicationWorker.java

```diff
@@ -537,21 +537,21 @@ public void testRWShutdownOnLocalBookieReadonlyTransition() throws Exception {
             bsConfs.get(bsConfs.size() - 1).setReadOnlyModeEnabled(true);
```

Comment:
Can we rename the 'testRWShutdownOnLocalBookieReadonlyTransition' to 'testRWOnLocalBookieReadonlyTransition'